src/Utils/obiDialogParser.ts

```diff
@@ -78,7 +80,9 @@ export class ObiDialogParser {
         if (!mainDialog) {
             this.warnings.push(`Missing entry point. Expecting a .dialog file called "Entry.main"`)
         } else {
-            const rootNode = await this.collectDialogNodes(mainDialog)
+            let conditionalEntities: { [key: string]: Set<string> } = {}
```

Comment:
Seems like this should be `const`. I'm guessing you made it `let` because it was passed in to the `collectDialogNodes` but you likely didn't mean to indicate that the method will overwrite the `conditionEntities` to something like a new array. Yu likely meant that it would adjust the `Set`'s

Reply:
Done